### ISSUE TITLE
Crash report alpha-58: OOB in AztecText.showLinkDialog

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/AztecEditor-Android#354]
```
Fatal Exception: java.lang.StringIndexOutOfBoundsException
       at android.text.SpannableStringBuilder.<init>(SpannableStringBuilder.java:61)
       at android.text.SpannableStringBuilder.subSequence(SpannableStringBuilder.java:1194)
       at org.wordpress.aztec.AztecText.getSelectedText(AztecText.kt:491)
       at org.wordpress.aztec.formatting.LinkFormatter.getSelectedUrlWithAnchor(LinkFormatter.kt:31)
       at org.wordpress.aztec.AztecText.showLinkDialog(AztecText.kt:889)
       at org.wordpress.aztec.AztecText.showLinkDialog$default(AztecText.kt:888)
       at org.wordpress.aztec.toolbar.AztecToolbar.onToolbarAction(AztecToolbar.kt:339)
       at org.wordpress.aztec.toolbar.AztecToolbar.access$onToolbarAction(AztecToolbar.kt:23)
       at org.wordpress.aztec.toolbar.AztecToolbar$initView$1.onClick(AztecToolbar.kt:266)
       at android.view.View.performClick(View.java:5637)
       at android.widget.CompoundButton.performClick(CompoundButton.java:122)
       at android.view.View$PerformClick.run(View.java:22429)
       at android.os.Handler.handleCallback(Handler.java:751)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6121)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

**Comment by maxme** [comment]
Ref https://github.com/wordpress-mobile/AztecEditor-Android/issues/265 and the fix https://github.com/wordpress-mobile/AztecEditor-Android/pull/272

**Comment by khaykov** [comment]
Fixed (hopefully) in #375
